### PR TITLE
Domain transfer tweaks

### PR DESCRIPTION
**Pull request by benrudolph**
Using this as a PR to address all QA comments

### CODE REVIEW

File: corehq/apps/domain/models.py [hunk 1]
```diff
@@ -1209,6 +1209,9 @@ def transfer_domain(self, *args, **kwargs):
         html_content = render_to_string(
             "{template}.html".format(template=self.DIMAGI_CONFIRM_EMAIL),
             self.as_dict())
+        text_content = render_to_string(
```
Comment (dimagimon) [inline corehq/apps/domain/models.py]: F841 local variable 'text_content' is assigned to but never used

File: corehq/apps/domain/templates/domain/activate_transfer_domain.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
       {% if transfer %}
         <p class="lead">
           {% blocktrans with domain=transfer.domain%}
-          By clicking "accept" below you acknowledge that you accept full ownership of this project space ("{{ domain }}").  You agree to be bound by the terms of Dimagi's <a href="https://www.commcarehq.org/eula/">End User License Agreement</a>.  By accepting this agreement, your are acknowledging you have permission and authority to accept Dimagi's End User License Agreement.
+          By clicking "accept" below You acknowledge that You accept full ownership of this project space ("{{ domain }}").  You agree to be bound by the terms of Dimagi's <a href="https://www.commcarehq.org/eula/">End User License Agreement</a>.  By accepting this agreement, your are acknowledging you have permission and authority to accept Dimagi's End User License Agreement. A Dimagi representative will notify you when the transfer is complete.
```
Comment (czue) [inline corehq/apps/domain/templates/domain/activate_transfer_domain.html]: part of the QA feedback was to improperly apply english capitalization rules?

**Comment by czue** [comment]
looks good - not merging because of label

File: corehq/apps/domain/templates/domain/activate_transfer_domain.html (hunk 2 above)
Comment (benrudolph) [inline corehq/apps/domain/templates/domain/activate_transfer_domain.html]: yea, i really think those should stay lower case. the actual spec has the text and they're capitalized in that (as well as the extra sentence that was added). wasn't sure if it was some sort of legal jargon

**Pull request merged**